src/components/BottomBar/BottomBarSagas.js

```diff
@@ -25,6 +25,7 @@ function* handleChangeParentButtonClick({ data: { note } }) {
 function* deleteNote({ data: { note } }) {
   yield noteStorage.remove(note);
   yield put(deleteNoteRequestSuccessAction(note));
+  yield put(push('/notes'));
```

Comment:
I believe this redirect is not needed, as `BottomBar` is shown on the `/notes` page.